x-pack/plugins/fleet/server/routes/security.ts

```diff
@@ -84,14 +84,20 @@ export async function getAuthzFromRequest(req: KibanaRequest): Promise<FleetAuth
 
     return calculateAuthz({
       fleet: { all: fleetAllAuth, setup: fleetSetupAuth },
-      integrations: { all: intAllAuth, read: intReadAuth },
+      integrations: {
+        all: intAllAuth,
+        read: intReadAuth,
+      },
       isSuperuser: checkSuperuser(req),
     });
   }
 
   return calculateAuthz({
     fleet: { all: false, setup: false },
-    integrations: { all: false, read: false },
+    integrations: {
+      all: false,
+      read: false,
+    },
```

Comment:
nit: Maybe also manually indent the `fleet` property to have new lines, if you meant to do this for readability purpose for `integrations`